packages/osd-optimizer/src/common/bundle.test.ts

```diff
@@ -88,13 +89,16 @@ it('provides the module count from the cache', () => {
 
 it('parses bundles from JSON specs', () => {
   const bundles = parseBundles(JSON.stringify([SPEC]));
+  let expectedCachePath = resolve('/foo/bar/target/.osd-optimizer-cache');
+  // Cannot use `standardize` from `@osd/util` due to mocking of fs
+  if (process?.platform === 'win32') expectedCachePath = expectedCachePath.replace(/\\/g, '\\\\');
```

Comment:
do we need a `?` here?

Reply:
I figured it is common so it could be called in unit tests as well as integration tests; just being safe.